[2024-08-21 15:23:34,728] 24 root - INFO - This is an informational message.
[2024-08-21 15:23:34,728] 25 root - ERROR - This is an error message.
[2024-08-21 15:23:46,718] 18 root - INFO - data loading started...
[2024-08-21 15:23:46,825] 21 root - INFO - data loading completed...
[2024-08-21 15:23:50,972] 44 root - ERROR - An error occurred: Error occured in python script name [C:\Users\DELL\qasystemnew\QAWithPDF\model_api.py] line number [27] error message [403 Request had insufficient authentication scopes. [reason: "ACCESS_TOKEN_SCOPE_INSUFFICIENT"
domain: "googleapis.com"
metadata {
  key: "service"
  value: "generativelanguage.googleapis.com"
}
metadata {
  key: "method"
  value: "google.ai.generativelanguage.v1beta.ModelService.GetModel"
}
]]
